src/backends/kimchi/mod.rs

```diff
@@ -133,58 +136,11 @@ impl KimchiVesta {
             pending_generic_gate: None,
             debug_info: vec![],
             finalized: false,
+            public_input_size: 0,
```

Comment:
would it make sense to pass that as argument in `new()`?

Reply:
actually it doesn't make sense, rereading things, as it gets calculated when we call add_public_input and add_public_output

Reply:
yeah, that was wondering this fact